server/routes/auth.js

```diff
@@ -1,20 +1,23 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
-const { validateRegister, validateLogin } = require('../validate');
-const protect = require('../middleware/auth');
+const { validateRegister, validateLogin } = require("../validate");
+const protect = require("../middleware/auth");
 const {
   registerUser,
   loginUser,
   loadUser,
   logoutUser,
-} = require('../controllers/auth');
+} = require("../controllers/auth");
+const { populate } = require("../controllers/demo");
 
-router.route('/register').post(validateRegister, registerUser);
+router.route("/register").post(validateRegister, registerUser);
 
-router.route('/login').post(validateLogin, loginUser);
+router.route("/login").post(validateLogin, loginUser);
 
-router.route('/user').get(protect, loadUser);
+router.route("/user").get(protect, loadUser);
 
-router.route('/logout').get(logoutUser);
+router.route("/logout").get(logoutUser);
+
+router.route("/populate").post(populate);
```

Comment:
Is this route strictly for creating the demo user? We probably don't need a route for that, I think I get the idea here - we want to give an empty state for every login of the demo user. Could this be a potential option for a task scheduler to reset the demo account without having to do a whole database drop and then creating a demo account?

Reply:
I have removed the route. I feel having a task scheduler will be too much. I have designated seeding after database connection and used the login method instead on the client side. This has reduced code and still has the same effect.